### ISSUE TITLE
PDF Viewer search bar is partially hidden on small screens

### ISSUE THREAD

**Issue by dimodi** [telerik/kendo-themes#4968]
**Describe the bug**

The PDF Viewer search bar is not fully visible on small screens, especially if the PDF zoom is less than 100%.

Reported in ticket 1647069

**To reproduce**

1. Open https://www.telerik.com/kendo-react-ui/components/pdfviewer/ or https://demos.telerik.com/blazor-ui/pdfviewer/overview
1. Set the PDF zoom to 50%.
1. Open the search popup.
1. Reduce the browser viewport width.
1. The search popup will be partially hidden on the left.

**Expected behavior**

The search bar should be fully visible and the items inside should wrap is necessary.

**Screenshots**

![5919f7ea-a688-4fd2-8ab1-c424f872b735_telerikpdfviewermobile](https://github.com/telerik/kendo-themes/assets/961014/071021ab-0f84-4b6d-8580-945d7a85483e)

**Affected package (please remove the unneeded items)**
* theme-default
* theme-bootstrap
* theme-material

**Affected suites (please remove the unneeded items)**
* Kendo UI for Angular
* Kendo UI for React
* Kendo UI for Vue
* Telerik UI for Blazor

**Affected browsers (please remove the unneeded items)**
* All
